<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_scr_dish_dish_quit</class>
 <widget class="QDialog" name="w_scr_dish_dish_quit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QFrame" name="frame_dishdetail">
       <property name="outer_stylesheet" stdset="0">
        <string>frame_detail_show</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>可退数量</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>退货原因</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="lds_doublespinbox_lineedit" name="lineEdit_num_num">
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>已退数量</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>商品数量</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>退货数量</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="3">
         <widget class="QComboBox" name="comboBox_vch_reason">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>所退商品</string>
          </property>
          <property name="buddy">
           <cstring>frame_dishdetail</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="lds_doublespinbox_lineedit" name="lineEdit_num_back">
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="lds_doublespinbox_lineedit" name="lineEdit_num2back">
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="4" rowspan="4">
         <widget class="ftableview" name="tableView"/>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="QLineEdit" name="lineEdit_vch_dishname">
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="lds_doublespinbox" name="lineEdit_inputnum"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_all_quit">
       <property name="text">
        <string>整单退货</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_ok">
       <property name="text">
        <string>退货</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ftableview</class>
   <extends>QTableView</extends>
   <header>ftableview.h</header>
  </customwidget>
  <customwidget>
   <class>lds_toolbar_button</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button.h</header>
  </customwidget>
  <customwidget>
   <class>lds_doublespinbox</class>
   <extends>QDoubleSpinBox</extends>
   <header>lds_doublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>lds_doublespinbox_lineedit</class>
   <extends>QLineEdit</extends>
   <header>lds_doublespinbox_lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
